src/psyclone/transformations.py

```diff
@@ -1197,14 +1223,14 @@ def apply(self, node):
         else:  # no distributed memory
             colour_loop.set_upper_bound("ncolour")
         # Add this loop as a child of our loop over colours
-        colours_loop.addchild(colour_loop)
+        colours_loop.loop_body.addchild(colour_loop)
 
         # add contents of node to colour loop
-        colour_loop.children.extend(node.children)
+        colour_loop.loop_body._children.extend(node.loop_body)
```

Comment:
Again, can we replace `_children` with `children` here?

Reply:
Done